tests/codegen/naked-fn/aligned.rs

```diff
@@ -0,0 +1,20 @@
+//@ compile-flags: -C no-prepopulate-passes -Copt-level=0
+//@ needs-asm-support
+//@ only-x86_64
+
+#![crate_type = "lib"]
+#![feature(naked_functions, fn_align)]
+use std::arch::asm;
+
+// CHECK: Function Attrs: naked
```

Comment:
Hmm, this is just checking a comment but that's probably fine since the check that validates the attribute is actually attached to the function is more annoying to write and LLVM doesn't emit these comments for no reason.

Reply:
this is similar to how the tests for naked functions itself work. It may not be perfect but at least it's consistent as-is